### PR TITLE
Main section

### PR DESCRIPTION
**Pull request by ElenaRise**

### CODE REVIEW

**Comment by vaov1999** [comment]
it's just comment

File: src/styles/index.scss [hunk 1]
```diff
@@ -30,6 +30,18 @@ body {
   border-bottom: 1px solid $color-firm10;
 }
 
+.app__section--main {
```
Comment (Alexei-Kekin) [inline src/styles/index.scss]: --main its modification or section?
if its section I think better write: .app__section main or .app__section main_section

File: src/styles/typography.scss [hunk 2]
```diff
@@ -2,6 +2,11 @@
...
 h1 {
   font-size: $font-size1;
+  text-transform: uppercase;
+  font-weight: 700;
+  letter-spacing: 0px;
+  margin: 0 auto;
+  color: $color-firm2;
 }
 
 h2 {
```
Comment (TonyXdddd) [inline src/styles/typography.scss]: need a small team of cleaners :)

Comment (TonyXdddd) [reply src/styles/typography.scss]: to delete "h2".

But you are so good !Well done bro:D

**Pull request closed without merging**